common/modal.js

```diff
@@ -383,12 +384,7 @@
             switch (params.selectMode) {
                 case modalBox.multiSelectMode:
                     if (params.onSelectedRowsChanged) {
-                        var res = params.onSelectedRowsChanged(getMultiSelectionResult());
-
-                        // if it returns false, then we should disable submit button
-                        // since we cannot apply the change (url limit issue)
-                        vm.disableSubmit = (res === false);
-                        return res;
+                        return params.onSelectedRowsChanged(getMultiSelectionResult());
```

Comment:
As we discussed, you should revert this to what it was since we want the button to be disabled when users reach URL length limitation while changing the selected rows in facet popup.